### PR TITLE
add shooter(fake) subtitle

### PR DESCRIPTION
**Pull request by fffonion**
redirect SearchByTitle to sub.makedie.me

### CODE REVIEW

File: service.subtitles.makedie/service.py [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+    if xbmcvfs.exists(__temp__):
+        rmtree(__temp__)
+    xbmc.sleep(50)
+    xbmcvfs.mkdirs(__temp__)
+
+    if item['mansearch']:
+        title = item['mansearchstr']
+    else:
+        title = '%s %s' % (item['title'], item['year'])
+        getSubByTitle(title, item['3let_language'])
```
Comment (taxigps) [inline service.subtitles.makedie/service.py]: call getSubByTitle in else statment will let it not work when user enter a search string.

**Comment by fffonion** [comment]
sorry for multiple commits :)

**Comment by fffonion** [comment]
Now add a setting so there's always one search request.
PS:It seems I accidently delete local commit history and failed to rebase

**Comment by taxigps** [comment]
1. I just advice to change code like:
```
    if item['mansearch']: 
        title = item['mansearchstr'] 
    else: 
        title = '%s %s' % (item['title'], item['year']) 
        if 'chi' in item['3let_language']: 
            getSubByHash(item['file_original_path'], "chn", "zh", "Chinese") 
        if 'eng' in item['3let_language']: 
            getSubByHash(item['file_original_path'], "eng", "en", "English") 
    getSubByTitle(title, item['3let_language']) 
```
not mean add an setting. I think alway get sub by title is better.
2. see here about rebase -i:
http://blog.csdn.net/zmyde2010/article/details/8603810

**Comment by fffonion** [comment]
I add this setting because I'm suffering from very slow(5s+) search response from shooter player API. And for most of time, shooter(fake) can provide good enough result.
What about a setting to turn off API search and is on by default? This will not affect searchByTitle but add extra requests when it's turned on, for those who can't find sub from shooter(fake).

**Comment by fffonion** [comment]
now merged as one commit

**Comment by taxigps** [comment]
can't merge pull request because taxigps/xbmc-addons-chinese changed, need rebase your branch.

**Comment by fffonion** [comment]
now automatically merging works

**Comment by taxigps** [comment]
This plug-in show incompatible in repo. I think cause by requires of script.module.html5lib version 0.1.0.

**Comment by fffonion** [comment]
What version or platform does this take place? I've tested on lastest Kodi for Windows, Android and iOS and all works fine.
By the way is the version means minimun version or exact version it requiring? html5lib seems only have one version history in the repo.

**Comment by taxigps** [comment]
test again, it's ok now.

**Comment by fffonion** [comment]
good

**Pull request merged**